iml-agent/src/action_plugins/stratagem/action_warning.rs

```diff
@@ -3,6 +3,7 @@
 // license that can be found in the LICENSE file.
 
 use crate::{
+    action_plugins::stratagem::util::search_rootpath,
```

Comment:
Wouldn't this be better suited in `lustre.rs`? Working with fids isn't inherently stratagem specific